client/me/two-step/index.jsx

```diff
@@ -14,92 +14,40 @@ import AppPasswords from 'calypso/me/application-passwords';
 import { Card } from '@automattic/components';
 import config from 'calypso/config';
 import DocumentHead from 'calypso/components/data/document-head';
+import { fetchUserSettings } from 'calypso/state/user-settings/actions';
```

Comment:
Nit: while we're here, wanna sort these imports alphabetically? Definitely consider that an optional improvement and not a blocker.

Reply:
Yup https://github.com/Automattic/wp-calypso/pull/49002/commits/cdb82354bc65a364f049b80d35a1bcc7321d5ae4